docs/how-to/migrate/wagtail-modeltranslation.md

```diff
@@ -0,0 +1,78 @@
+# Migrate from wagtail-modeltranslation
+
+
+### Remove wagtail-modeltranslation
+
+
+The package wagtail-modeltranslation, when installed and used, will add a new column in your database for every field
+you have.
+
+So if you have:
+```
+class MyPage(Page):
+    custom_field = models.CharField(...)
+```
+
+It will generate `custom_field_en`, `custom_field_de`, `custom_field_fr`, and so on for every field in your page.
+
+We need to remove the package and then remove the fields to clear out the database.
+
+!!! note "Careful with production data"
+
+    Make sure you run these steps in a testing environment first. Once the fields (above) are deleted from your
```

Comment:
Do you think it would be possible to generate locale trees using the translations already created for the modeltranslation site?

Reply:
I am currently working on migrating a production site from `wagtail-modeltranslation` to `wagtail-localize` and might write up what needs to be done to migrate the existing translations.

Reply:
That is great to hear @saevarom! Looking forward to it

Reply:
so... how did it turn out 😛 ?